From 1758c5d55aff7fb8c97695f096b75e1594f3020a Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Thu, 16 Oct 2014 16:02:06 -0400
Subject: [scsi] mpt3sas: Bump mpt3sas driver version to 03.100.00.00

Message-id: <1413475335-14737-24-git-send-email-thenzl@redhat.com>
Patchwork-id: 97615
O-Subject: [RHEL7.1 e-stor PATCH 23/32] mpt3sas: Bump mpt3sas driver version to 03.100.00.00
Bugzilla: 1088524
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

Bump mpt3sas driver version to 03.100.00.00.

Signed-off-by: Sreekanth Reddy <Sreekanth.Reddy@avagotech.com>
Reviewed-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit bd0a791ce281062f29ac984dcb64c960e24d2b0d)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/mpt3sas/mpt3sas_base.h b/drivers/scsi/mpt3sas/mpt3sas_base.h
index 184826c..575f1bc 100644
--- a/drivers/scsi/mpt3sas/mpt3sas_base.h
+++ b/drivers/scsi/mpt3sas/mpt3sas_base.h
@@ -70,8 +70,8 @@
 #define MPT3SAS_DRIVER_NAME  "mpt3sas"
 #define MPT3SAS_AUTHOR "LSI Corporation <DL-MPTFusionLinux@lsi.com>"
 #define MPT3SAS_DESCRIPTION "LSI MPT Fusion SAS 3.0 Device Driver"
-#define MPT3SAS_DRIVER_VERSION  "02.100.00.00"
-#define MPT3SAS_MAJOR_VERSION  2
+#define MPT3SAS_DRIVER_VERSION  "03.100.00.00"
+#define MPT3SAS_MAJOR_VERSION  3
 #define MPT3SAS_MINOR_VERSION  100
 #define MPT3SAS_BUILD_VERSION  0
 #define MPT3SAS_RELEASE_VERSION 00
-- 
1.7.1